### DISCUSSION TITLE
Output parte 1

### DISCUSSION THREAD

**Discussion by adribastiasm** [IIC2133-PUC/2024-1#77] [Tarea 1] [answered]
Hola! 
Tengo una duda respecto al output de la parte 1 de la tarea, en los tests 7 y 8.
El output esperado en una parte es: "Mercado inactivo: No hay registros de transacciones"
y yo lo tengo con un punto final porque eso sale en el enunciado. 
Debería dejarlo con punto o sin punto?
En todo caso no he revisado los testcases nuevos que subieron porque me dice que necesito acceso, por si pueden ayudarme con eso también.
Estaré atenta, saludoss!

**Comment by benjavicente** [comment, marked as answer]
Hola! Está actualizado ese caso. Es el mismo link de los otros anuncios de las tareas (va a ser el mismo a lo largo del semestre), [lo dejé aquí en GitHub también](https://github.com/IIC2133-PUC/2024-1/tree/main/Tareas). Pronto mando un anuncio con el link correcto.

**Reply by adribastiasm** [reply]
Ahí pude abrir el link, muchas gracias!
